sites/example-project/src/components/ui/PageMenu.svelte

```diff
@@ -27,14 +32,38 @@
 	function toggleDropdown() {
 		showDropdown = !showDropdown;
 	}
+
+	// Copied from https://github.com/svelteuidev/svelteui/blob/main/packages/svelteui-composables/src/actions/use-click-outside/use-click-outside.ts
+	function clickoutside(node, params) {
+		const { enabled: initialEnabled, callback } = params;
+
+		const handleOutsideClick = ({ target }) => {
+			if (!node.contains(target)) callback(node);
+		};
+
+		function update({ enabled }) {
+			if (enabled) {
+				window.addEventListener('click', handleOutsideClick);
+			} else {
+				window.removeEventListener('click', handleOutsideClick);
+			}
+		}
+		update({ enabled: initialEnabled });
+		return {
+			update,
+			destroy() {
+				window.removeEventListener('click', handleOutsideClick);
+			}
+		};
+	}
 </script>
```

Comment:
This should get moved to it's own file; it's something that we are probably going to want to use in other places.